src/invidious/helpers/crystal_class_overrides.cr

```diff
@@ -18,6 +18,40 @@ end
 class HTTP::Client
   property family : Socket::Family = Socket::Family::UNSPEC
 
+  # Override stdlib to automatically initialize proxy if configured
```

Comment:
On second thought, I think it'd be better to just initialize the proxy on each http request rather than override stdlib.

Reply:
Yes, It would be better indeed!